### ISSUE TITLE
Make get_plotlyjs public

### ISSUE THREAD

**Issue by PhML** [plotly/plotly.py#637]
From the plot docstring:

```
include_plotlyjs (default=True) -- If True, include the plotly.js
    source code in the output file or string.
    Set as False if your HTML file already contains a copy of the plotly.js
    library.
```
This option is not really useful if we can not get the library in the HTML file in first place. It is just a matter of adding `get_plotlyjs` in the `__init__` file of the offline module.

**Comment by jonmmease** [comment]
Thanks for the suggestion @PhML, this is being added for version 3.4.0 in #1246

**Comment by jonmmease** [comment]
Implemented in #1246
